downstream/modules/platform/ref-general-inventory-variables.adoc

```diff
@@ -26,6 +26,9 @@ Default = `TLSv1.2`
 Each element in the list is provided into `http nginx config` as a separate line. 
 
 Default = empty list
+| `redis_mode` | | Redis can be colocated with any other node in a clustered installation. 
```

Comment:
Redis cannot be colocated with automation controller, automation controller execution, or database nodes

Reply:
They can only be automationgateway, automationhub, automationedacontroller nodes

Reply:
Gotcha. Updated